pandas/io/formats/format.py

```diff
@@ -599,7 +599,7 @@ def to_string(self):
             else:  # max_cols == 0. Try to fit frame to terminal
                 text = self.adj.adjoin(1, *strcols).split('\n')
                 row_lens = Series(text).apply(len)
```

Comment:
shouldn't this just be

``max_len = Series(text).str.len().max()`` ?

Reply:
Tests seem to pass locally with this.